src/components/Global/TokenSelector/TokenSelector.tsx

```diff
@@ -134,33 +252,28 @@ const TokenSelector = ({ classNameButton, shouldBeConnected = true, onReset }: _
                     setVisible(!visible)
                 }}
                 isVisible={visible}
-                tokenLogoUri={(IconPlaceholderChecker(selectedChainID) as string) ?? displayedToken?.logoURI}
-                tokenSymbol={displayedToken?.symbol ?? ''}
-                tokenBalance={displayedTokenBalance?.amount ?? 0}
-                chainIconUri={(IconPlaceholderChecker(selectedChainID) as string) ?? displayedChain?.icon.url}
+                tokenLogoUri={selectedBalance?.logoURI ?? ''}
```

Comment:
question: what if the logoUri is empty? Then we get a broken looking icon

Can we use the fallback icon we have alrdy, like in this example:
![image](https://github.com/user-attachments/assets/706b9d81-f468-4810-bcf5-b0e1367b54fa)

Reply:
good idea, added